.travis.yml

```diff
@@ -1,22 +1,22 @@
 language: android
-
 android:
   components:
-    - build-tools-28.0.3
-    - android-28
-
-jobs:
+  - build-tools-28.0.3
+  - android-28
+jobs: 
   include:
-    - stage: compile
-      language: android
-      before_script: chmod +x gradlew
-      script:
-        - ./gradlew ktlint
-        - ./gradlew assembleDebug
-
-    - stage: tests
-      language: android
-      before_script: chmod +x gradlew
-      script:
-        - ./gradlew jacocoTestReport
-        - cd $TRAVIS_BUILD_DIR/displayJacoco/ && python output_jacoco_metrics.py
+  - stage: Linter
+    language: android
+    before_script: chmod +x gradlew
+    script:
+    - ./gradlew ktlint | python AnalysisCI/analyze_ci.py
```

Comment:
python command refers to which version of python?

Reply:
2.7